daemon/algod/api/algod.oas2.json

```diff
@@ -245,7 +245,7 @@
     },
     "/v2/accounts/{address}": {
```

Comment:
Recalling: we decided _not_ to put any heartbeat specific endpoints on the algod API right?

Reply:
Correct. At some point, and API that allowed an outside caller to create a heartbeat might be useful.  We'd have to decide how it should work, since presumably this would be a heartbeat the caller is paying for, so it needs to be signed. OTOH, only algod has the part keys, so it also needs to create a kind of signature.